site/src/main/scala/scalatex/site/Styles.scala

```diff
@@ -1,9 +1,9 @@
 package scalatex.site
 import scalatags.Text.all._
-import scalatags.stylesheet.{CascadingStyleSheet, Sheet}
+import scalatags.stylesheet.{CascadingStyleSheet, SourceClasses}
 
 object Styles{
-  val css = Sheet[Styles]
+  val css = new Styles {}
```

Comment:
Not free now, will try to find some time to take a look at this today or
tomorrow. The syntax for Styles did change; you no longer need the trait
T/Sheet[T] pair, and you can just have a single object inheriting from
stylesheet and defining cls-s. The Scalatags docs should have an example of
what the syntax looks like right now
http://www.lihaoyi.com/scalatags/#CSSStylesheets

On Tue, Aug 23, 2016 at 10:56 PM, Romain Reuillon <notifications@github.com>
wrote:

> In site/src/main/scala/scalatex/site/Styles.scala
> <https://github.com/lihaoyi/Scalatex/pull/30#discussion_r75883601>:
>
> >
> >  object Styles{
> > -  val css = Sheet[Styles]
> > +  val css = new Styles {}
>
> I tried witch SourceClasses as well but I was not able to make it compile.
> Is it correct?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lihaoyi/Scalatex/pull/30/files/236f126c4c0894724017deaeecab3617e6f64059#r75883601>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA5A_It9VL23elf_bHMCaRFkgu6aPd2pks5qiwojgaJpZM4JrBNE>
> .
>